src/javaharness/java/arcs/api/Arcs.java

```diff
@@ -0,0 +1,17 @@
+package arcs.api;
+
+public interface Arcs {
```

Comment:
I'm a little concerned that having both `Arc` and `Arcs` classes might be a little confusing. I wonder if there's a better name for one/both of them? Arc -> ArcData? Arcs -> ArcsRunner? Do you have any ideas?

Reply:
renamed Arc to ArcData